README.md

```diff
@@ -505,7 +505,11 @@ Default: False
 Set to true to disable fqdn SSL cert check
 Default: False
 
-##### `service_manage`
+#####`ssl_mode`
```

Comment:
Could you add a space? I believe this used to be allowed by GH markdown but they now follow CommonMark a bit stricter.